src/app/api/file/[fileId]/route.ts

```diff
@@ -58,8 +58,10 @@ export const PATCH = withSession(async ({ params, session, req }) => {
         { status: 400 }
       );
     }
-
-    const formatted_date = moment(fileData.date).format("L");
+    const newDate = new Date(
+      fileData.date.getTime() + new Date().getTimezoneOffset() * 60000
```

Comment:
Should we extract the logic into a helper function, since it's reused quite a bit?

Reply:
Yea definitely needed, added!